pkg/server/chaosd/disk.go

```diff
@@ -17,220 +17,111 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"path/filepath"
-	"strconv"
-	"strings"
 	"sync"
+	"time"
 
 	"github.com/hashicorp/go-multierror"
-	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
 	"go.uber.org/zap"
 
+	"github.com/chaos-mesh/chaosd/pkg/server/utils"
+
 	"github.com/chaos-mesh/chaosd/pkg/core"
-	"github.com/chaos-mesh/chaosd/pkg/utils"
 )
 
 type diskAttack struct{}
 
 var DiskAttack AttackType = diskAttack{}
 
-const DDWritePayloadCommand = "dd if=/dev/zero of=%s bs=%s count=%s oflag=dsync"
-const DDReadPayloadCommand = "dd if=%s of=/dev/null bs=%s count=%s iflag=dsync,fullblock,nocache"
-
-func (disk diskAttack) Attack(options core.AttackConfig, env Environment) (err error) {
-	attack := options.(*core.DiskOption)
+func (disk diskAttack) Attack(options core.AttackConfig, env Environment) error {
+	if attackConf, ok := options.(*core.DiskAttackConfig); ok {
```

Comment:
how about using `!ok` here and then return error, so we can save some indentation

Reply:
OK

Reply:
fixed.